lib/widgets/matrix.dart

```diff
@@ -424,6 +426,11 @@ class MatrixState extends State<Matrix> with WidgetsBindingObserver {
     if (kIsWeb) {
       onFocusSub = html.window.onFocus.listen((_) => webHasFocus = true);
       onBlurSub = html.window.onBlur.listen((_) => webHasFocus = false);
+      onClosingTab = html.window.onBeforeUnload.listen((_) {
+        if (client.isLogged()) {
+          _setOfflinePresence();
+        }
```

Comment:
@imGok you mean this function was not executed successfully?

Reply:
![signal-2023-09-25-110939_002](https://github.com/linagora/twake-on-matrix/assets/12546908/02e84dc8-7743-4b3b-ac0d-bbe65dcaf461)
https://spec.matrix.org/v1.8/client-server-api/#last-active-ago
I just read the spec
Seem like setOfflinePresence() isn't resolve the root cause because the server set user offline after a time. what do you think @imGok

Reply:
IMO it does because I managed to set offline when the user puts the app in background. I just couldn't put offline on closing app

Reply:
yes, agree